packages/instrumentation-typeorm/src/typeorm.ts

```diff
@@ -201,6 +202,7 @@ export class TypeormInstrumentation extends InstrumentationBase<typeof typeorm>
                     [SemanticAttributes.DB_OPERATION]: operation,
                     [SemanticAttributes.DB_STATEMENT]: sql,
                     [SemanticAttributes.DB_SQL_TABLE]: mainTableName,
+                    [ExtendedDatabaseAttribute.DB_STATEMENT_PARAMETERS]: JSON.stringify(parameters),
```

Comment:
As this is not in the spec, might contain sensitive information, and can potentially contain large size payloads, I think it's better to add it in a hook, so user can choose a strategy for how to handle it.
WDYT?

Reply:
I think the user expects to see his SQL query parameters in the instrumentation by default. and that's also what our customers expect and the motivation for this PR